buckaroo3.php

```diff
@@ -283,7 +283,7 @@ public function getContent()
         $jwt = new JWTAuth();
         $token = $this->generateToken($jwt);
```

Comment:
just realized, why do we need authentication in the admin area? isn't authentication provided by prestashop by default?